fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1026,11 +1060,14 @@ private FHIRPersistenceNotSupportedException buildNotSupportedException(String m
         final String METHODNAME = "delete";
         log.entering(CLASSNAME, METHODNAME);
 
-
+        // TODO this implementation needs to be updated so that it doesn't
+        // modify the resource - that now has to be done at the REST layer
```

Comment:
do we have a new flavor of the delete method for this already?  if not, whats to plan to resolve this one?

Reply:
Yes, agreed should be removed for 5.0.0

Reply:
https://github.com/IBM/FHIR/issues/3158